test/migration.go

```diff
@@ -78,3 +81,65 @@ func RunMigrationContainer(mysql *MySQLContainer, options MigrationOptions) erro
 
 	return container.Stop()
 }
+
+// RunGOMigrationContainer creates the migration container and connects to the
+// sql database container to run the migration scripts.
+func RunGoMigrationContainer(sql *SQLContainer, options MigrationOptions) error {
```

Comment:
It's more like RunRailsMigrationContainer because we pass the rake command to the container ? or did we implement it in Go?